source/kraken/apply_disruption.cpp

```diff
@@ -190,11 +202,94 @@ struct add_impacts_visitor : public apply_impacts_visitor {
     }
 
     void operator()(nt::StopPoint* stop_point) {
-        LOG4CPLUS_INFO(log, "Disruption on stop point:" << stop_point->uri << " is not handled");
+
+        using namespace boost::posix_time;
+        using namespace boost::gregorian;
+
+        type::ValidityPattern impact_vp{meta.production_date.begin()}; // bitset are all initialised to 0
+        for (const auto& period: impact->application_periods) {
+            auto one_day = boost::posix_time::time_duration{24, 0, 0};
+            for (time_iterator it(period.begin() - one_day, one_day); it < period.end(); ++it) {
+                if (! meta.production_date.contains((*it).date())) { continue; }
+                auto day = ((*it).date() - meta.production_date.begin()).days();
+                impact_vp.add(day);
+            }
+        }
+
+        std::vector<std::pair<const nt::VehicleJourney*, nt::ValidityPattern>> vj_vp_pairs;
```

Comment:
Comment : `Determine list of concerned vjs, and gives their "base_canceled" vp (hierarchical vp)`